client/src/components/Page/PageDropdown.vue

```diff
@@ -57,19 +62,31 @@ function onDelete(page_id: string) {
         </b-link>
         <p v-if="props.page.description">{{ props.page.description }}</p>
         <div class="dropdown-menu" aria-labelledby="page-dropdown">
-            <a class="dropdown-item dropdown-item-view" :href="urlView">
+            <a class="dropdown-item dropdown-item-view" href="javascript:void(0)" @click="pushUrl(urlView)">
```

Comment:
Thank you for refining this. We could use `router-link` here like we do e.g. in the admin panel: `<router-link to="/admin/sanitize_allow">Manage Allowlist</router-link>`. Then right click on links would work and the `computed` functions and then `pushUrl` helper would not be needed.

Reply:
ohoho! nice, I'll do that